### DISCUSSION TITLE
Copy image as reference / link to local path to paste it in another page without copying the file asset itself

### DISCUSSION THREAD

**Discussion by hsandt** [zim-desktop-wiki/zim-desktop-wiki#2442] [Ideas]
I often refer to the same entry (here, a video game title) in multiple pages, and I like to add some screenshot or thumbnail to visualize what it is.

Currently right-click > Copy or highlight image area > Copy will copy and paste a brand new copy of the image. It's great when you want to be sure to have a local copy of the image besides the note page, to avoid relying on a different page and its assets, which may be removed at some point and break the new page.

However, to spare space, I also like relinking the same pictures in my notes. Sometimes, I even link to external files I know I'll have on every computer, such as Dropbox files.

In this case, it makes more sense to copy a *reference* to the image, without creating any new asset. The image would be from another page, actually stored in a Zim notebook, or an external image somewhere on disk.

This idea is similar to https://github.com/zim-desktop-wiki/zim-desktop-wiki/issues/884 except you'd fetch the image from a local URI as usual, not a web URL.

Currently, Zim can already show arbitrary images on disk, so a workaround is possible:
1. right-click on image to copy reference of > Open Folder
2. open page to paste image reference to
3. drag-and-drop image from file browser opened in step 1, into the page

**Comment by introt** [comment]
Related FR: #1937
